server/src/db/migrations/20210101210207_createUsers.cjs

```diff
@@ -15,6 +15,7 @@ exports.up = async (knex) => {
     console.log(`Creating ${tableName}`);
     return knex.schema.createTable(tableName, (table) => {
       table.bigIncrements("id");
+      table.string("name").notNullable().unique()
```

Comment:
Might not want the name to be unique - typically people would enter their real name which means there can't be 2 "Nick"'s. Instead, if you want uniqueness you can update this to a `username` which is usually unique